earth2mip/initial_conditions/gfs.py

```diff
@@ -224,7 +228,10 @@ def get(
         data[i] = field
 
     # Clean up
-    shutil.rmtree(GFS_CACHE)
+    if download_gfs:
+        shutil.rmtree(GFS_CACHE)
```

Comment:
Might be nice to log that it's getting deleted, just to distinguish it from downloading IC.

Reply:
Added in, thanks for the suggestion.